index.js

```diff
@@ -105,7 +107,7 @@ module.exports = ({utPort}) => class SchedulePort extends utPort {
             }
             extLoadInterval = extLoadInterval * 1000;
 
-            setInterval(this.extLoad.bind(this), extLoadInterval);
+            this.interval = setInterval(this.extLoad.bind(this), extLoadInterval);
```

Comment:
before this line, we can check just in case, if there was a previous attempt to start and clear the interval:
```js
if (this.interval) clearInterval(this.interval);
```